### DISCUSSION TITLE
Can't use custom prefixes

### DISCUSSION THREAD

**Discussion by Tamo73** [jagrosh/MusicBot#1211] [Troubleshooting] [answered]
I changed my custom prefix to "-" a while ago and it worked fine until today, and only the "@mention" prefix is working.
I tried to change the default prefix and it didn't work either, even when I use other alt prefixes (I tried "+", "*" and "!" as alt prefixes). I also tried changing the default prefix directly from the config.txt file, but it stopped working with anything but the default "@mention".
I attached an image showing the problem.
Thanks for the help!
![01211942](https://user-images.githubusercontent.com/104340634/188039373-b66b98ef-1930-4759-9df9-113d970e94df.png)

**Comment by jagrosh** [comment, marked as answer]
https://github.com/jagrosh/MusicBot/issues/1207

**Reply by Tamo73** [reply]
Weirdly enough, the option was on, but after toggling it off and on and restarting discord it worked fine again. Thanks!
